### ISSUE TITLE
Ruby on Rails Course: Add tips for using Rails short-cut commands

### ISSUE THREAD

**Issue by Overlandcoder** [TheOdinProject/curriculum#24919]
### Describe your suggestion

There are a few handy short-cut Rails commands that could be listed in either the Micro-Reddit project or the Forms project. The main short-cut commands that I know of: `c` for `console`, `s` for `server`, `g` for `generate`.

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-on-rails-micro-reddit

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Overlander

### (Optional) Additional Comments

I researched to see if there are more similar shortcuts, but couldn't find any (at least not ones that would be useful for students at that stage).

Related to #24885

**Comment by CouchofTomato** [comment]
@Overlandcoder Are you happy to work on this? If yes what support do you need?

**Comment by ChargrilledChook** [comment]
Some ideas we discussed that could potentially be good to do as part of this (but not necessary - could be out of scope)

- Would it help to have a section or lesson on the Rails command line more generally?
- Would it help to have a section explaining bin scripts? Both what a binstub is, and what conveniences you get out of the box, possibly how to write your own. I believe @JoshDevHub has raised this as an idea before
- Would it help to have a section on writing your own aliases or shortcuts in bash / zsh?

**Comment by Overlandcoder** [comment]
I made a PR to add the tips in.

@ChargrilledChook My thinking is that it may overload the students with info, since the RoR path is already very dense. I've noticed that students risk burnout with chess, then go through a few marathon lessons on Forms, and then the new world of Rails overwhelms even the grittiest of students (personally noticed this with one or two students recently).

If those sections are added, I think it should be stated that they are optional. Curious to hear what others think though!
